src/frontend/magnify/apps/magnify-site/i18n/frontend.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Why are the translations in the vite commit ?

Reply:
I ran the command for translations to see if everything worked as before. Since I had to change the path